host/host_linux.go

```diff
@@ -73,14 +73,24 @@ func InfoWithContext(ctx context.Context) (*InfoStat, error) {
 	}
 
 	sysProductUUID := common.HostSys("class/dmi/id/product_uuid")
+	machineID := common.HostEtc("machine-id")
 	switch {
+	// When not running as root, lib cannot read the value
```

Comment:
This is not the only way gopsutil could not be able to read this value, the file can be missing for example (this is the case on arm on Debian 9). Remove this comment or completely rephrase it to explain what this `switch case fallthrough` construction does.